docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -7,45 +7,169 @@ This section provides instructions for migrating your hosts away from your old M
+   
+3. After installation, the MDM enrollment profile can be removed by the end user at any time.
+
+### How to turn on MDM
+
+1. Select the Fleet icon in your menu bar and select **My device**.
+
+![Fleet icon in menu bar](https://raw.githubusercontent.com/fleetdm/fleet/main/website/assets/images/articles/fleet-desktop-says-hello-world-cover-1600x900@2x.jpg)
+
+2. On your **My device** page, select **Turn on MDM** the button in the yellow banner and follow the instructions. 
+  - If you don’t see the yellow banner or the **Turn on MDM** button, select the purple **Refetch** button at the top of the page. 
+  - If you still don't see the **Turn on MDM** button or the **My device** page presents you with an error, please contact your IT administrator.
+
+<img width="1400" alt="My device page - turn on MDM" src="https://user-images.githubusercontent.com/5359586/229950406-98343bf7-9653-4117-a8f5-c03359ba0d86.png">
+
+## Migrate automatically enrolled (DEP) hosts
 
 > Automatic enrollment is available in Fleet Premium or Ultimate
 
+To migrate automatically enrolled hosts, we will do the following steps:
+
+1. Prepare to migrate hosts
+2. Choose migration workflow and migrate hosts
+
+### Step 1: prepare to migrate hosts
+
 1. Connect Fleet to Apple Business Manager (ABM). Learn how [here](./MDM-setup.md#apple-business-manager-abm).
 2. [Enroll](./Adding-hosts.md) your hosts to Fleet with [Fleetd and Fleet Desktop](https://fleetdm.com/docs/using-fleet/adding-hosts#including-fleet-desktop) 
 3. Ensure your end users have access to an admin account on their Mac. End users won't be able to migrate on their own if they have a standard account.
 4. Migrate your hosts to Fleet in ABM:
     1. In ABM, unassign the existing hosts' MDM server from the old MDM solution: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Unassign from the current MDM**, and select **Continue**.
     2. In ABM, assign these hosts' MDM server to Fleet: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Assign to the following MDM:**, select your Fleet server in the dropdown, and select **Continue**.
-5. In your old MDM solution, unenroll the hosts to be migrated. MacOS does not allow multiple MDMs to be installed at once.
-6. Send [these guided instructions](#instructions-for-end-users) to your end users to complete the final few steps via Fleet Desktop.
-    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old
-      MDM and when the host turns on MDM in Fleet. Use Fleet's [end user migration workflow](#end-user-migration-workflow) to reduce the gap in MDM coverage.
 
-### End user migration workflow
+### Step 2: choose migration workflow and migrate hosts
+
+There are two migration workflows in Fleet: default and end user.
```

Comment:
Hmm, yeah I was considering these options:
1. "IT initiated" and "End user initiated"
2. "Default workflow" and "End user workflow"
3. "Default" and "end user" (current)

I liked "IT initiated" and "End user initiated" because the difference between the workflows is more explicit. 

However, we call the end user migration workflow feature "end user migration workflow" in the product. We don't use the work "initiated."

I went with #3 because I think "default" tells the user that this will be the workflow used if they don't do any additional set up. "Default" is shorter than "Default workflow."

What do you think?
I aldo